cmsl1t/playground/eventreader.py

```diff
@@ -246,12 +253,20 @@ def __init__(self, files, events=-1):
             else:
                 input_files.append(f)
         # this is not efficient
-        self._trees = [TreeChain(name, input_files, cache=True, events=events)
-                       for name in TREE_NAMES]
+        self._trees = []
+        self._names = []
+        for name, path in ALL_TREE.iteritems():
+            try:
+                chain = TreeChain(path, input_files, cache=True, events=events)
+            except RuntimeError:
+                logger.warn("Cannot find tree: {0} in input file".format(path))
+                continue
+            self._names.append(name)
+            self._trees.append(chain)
 
     def __iter__(self):
-        for trees in six.moves.zip(*self._trees):
-            yield Event(trees)
+        for i, trees in enumerate(six.moves.zip(*self._trees)):
```

Comment:
I think it was from some debugging I had to do at one point. I'll take this out, although I'm surprised it wasn't caught by flake8. I suppose it's common to want to discard a part of a generator-returned tuple like this, so maybe that's why... Anyway, going...